pkg/apic/apiserver/models/api/v1/resourceinstance.go

```diff
@@ -9,7 +9,7 @@ import (
 type ResourceInstance struct {
 	ResourceMeta
 
-	Owner interface{} `json:"owner,omitempty"`
+	Owner *Owner `json:"owner,omitempty"`
```

Comment:
If the others are not omitempty, should this be? Should be consistent, no?

Reply:
I think maybe. I've been doing some reading and testing. I'll do it.